sdcm/sct_events/loaders.py

```diff
@@ -223,15 +224,22 @@ def msgfmt(self):
                                           regex="Using optimized driver")
 CassandraStressLogEvent.add_subevent_type("SchemaDisagreement", severity=Severity.WARNING,
                                           regex="No schema agreement")
-
+# By default audit is disabled in 20223.1 by https://github.com/scylladb/scylla-enterprise/pull/3094.
+# But it won't be disabled in 2022.1 and 2022.2.
+# This message generates too much noise for us. We do not need it will fail the test. Create WARNING message, not ERROR.
+# This event will be cretaed in branch 2023.1, not in 2022.x
+CassandraStressLogEvent.add_subevent_type("AuthenticationError", severity=Severity.WARNING,
```

Comment:
Now I might complicate things a bit...

maybe we can introduce it as a filter ? and put it in `enable_default_filters` like the rest ?
and also use `SkipPerIssue` ?

Reply:
There is no opened issue for that.

Reply:
@fruch 
I can add filter in `enable_default_filters` as :
```
DbEventsFilter(db_event=DatabaseLogEvent.BACKTRACE, line='Rate-limit: supressed').publish()
```

Reply:
> There is no opened issue for that. 
> 

There is but it's closed, we should get it tagged with the needed tag so it would be skip in 2023.1 and 2024.1

Reply:
> > There is no opened issue for that.
> 
> There is but it's closed, we should get it tagged with the needed tag so it would be skip in 2023.1 and 2024.1

Which tag do you mean? Label?

Reply:
> > There is no opened issue for that.
> 
> There is but it's closed, we should get it tagged with the needed tag so it would be skip in 2023.1 and 2024.1

@fruch  who can put the labels?

Reply:
As I see from the code, it should be label like 'sct-2023.1-skip'